### PR TITLE
Cherry pick of #283, #286, #294

### PR DESCRIPTION
**Pull request by keisukesakasai**
Cherry pick of https://github.com/kubernetes-sigs/hierarchical-namespaces/pull/283 , https://github.com/kubernetes-sigs/hierarchical-namespaces/pull/286 , https://github.com/kubernetes-sigs/hierarchical-namespaces/pull/294 on master.

https://github.com/kubernetes-sigs/hierarchical-namespaces/pull/283: Show HierarchicalResourceQuota status by kubectl-hns
https://github.com/kubernetes-sigs/hierarchical-namespaces/pull/286 : Remove unneeded dependency to k8s.io/kubernetes
https://github.com/kubernetes-sigs/hierarchical-namespaces/pull/294: Remove unneeded replace clause from go.mod

The discussion regarding the content can be found here:
https://github.com/kubernetes-sigs/hierarchical-namespaces/pull/284#issuecomment-1566706843

### CODE REVIEW

**Comment by keisukesakasai** [comment]
Hi, @adrianludwin 
Could you please check this issue, thanks.

/assign @adrianludwin

File: go.mod [hunk 1]
```diff
@@ -120,7 +119,34 @@ require (
...
 )
 
-require (
-	github.com/spf13/afero v1.6.0 // indirect
-	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20230208013708-22718275bffe // indirect
+require sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20230208013708-22718275bffe
+
+require github.com/spf13/afero v1.6.0 // indirect
+
+replace (
```
Comment (adrianludwin) [inline go.mod:126]: I didn't really question this last time - why is this here?

**Review by adrianludwin — COMMENTED** [review]
lgtm after one question about the go.mod changes

File: go.mod (hunk 1 above)
Comment (adrianludwin) [reply go.mod:126]: (The `replace` clause, just to clarify)

Comment (mochizuki875) [reply go.mod:126]: Hi, @adrianludwin 
In conclusion, the replace clause seems to be unnecessary.
It was added #283 as a workaround and has to be unneeded in #286.
Fundamentally this was my mistake... sorry.

So I removed it in #294 and please check it.

**Comment by rjbez17** [comment]
/ok-to-test
/lgtm

@adrianludwin can approve when ready

File: go.mod (hunk 1 above)
Comment (adrianludwin) [reply go.mod:126]: Thanks for fixing it, can you add #294 to this PR as well? Thanks!

Comment (keisukesakasai) [reply go.mod]: I did it, please check. Thanks!

Comment (mochizuki875) [reply go.mod]: @keisukesakasai Thanks your add:)
